Mathlib/Util/AddRelatedDecl.lean

```diff
@@ -66,6 +67,8 @@ def addRelatedDecl (src : Name) (suffix : String) (ref : Syntax)
   | _ => throwError "Constant {src} is not a theorem or definition."
   if isProtected (← getEnv) src then
     setEnv $ addProtected (← getEnv) tgt
-  if let some thisAttr := thisAttr then
-    let attrs := match attrs? with | some attrs => attrs | none => #[]
-    _ ← Term.TermElabM.run' <| ToAdditive.applyAttributes ref attrs thisAttr src tgt
+  let attrs := match attrs? with | some attrs => attrs | none => #[]
+  _ ← Term.TermElabM.run' <| do
+    let attrs ← elabAttrs attrs
+    Term.applyAttributes src attrs
+    Term.applyAttributes tgt attrs
```

Comment:
Untested:
```suggestion
  _ ← Term.TermElabM.run' <| do
    let attrs ← elabAttrs (attrs?.getD #[])
    Term.applyAttributes src attrs
    Term.applyAttributes tgt attrs
```

Reply:
There is actually a coercion on line 70, so this doesn't work